personality/xrootd/env.pan

```diff
@@ -35,3 +38,19 @@ variable XROOTD_MONITORING_DESTINATIONS ?= undef;
 variable XROOTD_REPORTING_DESTINATIONS ?= undef;
 
 
+# Default GSI-related parameters
+variable XROOTD_SECURITY_GSI_PARAMS ?= {
+  SELF["ca"] = 2;
+  SELF["cert"] = "/etc/grid-security/dpmmgr/dpmcert.pem";
+  SELF["crl"] = 3;
+  SELF["gmapopt"] = 10;
+  SELF["key"] = "/etc/grid-security/dpmmgr/dpmkey.pem";
+  SELF["md"] = "sha256:sha1";
+  if ( VOMS_XROOTD_EXTENSION_ENABLED ) { 
+    SELF["vomsfun"] = "/usr/lib64/libXrdSecgsiVOMS.so";
+  } else {
+    SELF["vomsat"] = 0;
+  };
+
+  SELF;
+};
```

Comment:
You are right! I'll try to update it later today... @Pansanel or @guillaume-philippon could you confirm that you don't see any problem merging it in 16.4. I would be important to allow a smooth deployment of https://github.com/quattor/configuration-modules-grid/pull/105 later.